prospero-common/src/main/java/org/wildfly/prospero/api/WellKnownRepositories.java

```diff
@@ -9,7 +9,7 @@
  */
 public enum WellKnownRepositories implements Supplier<RemoteRepository> {
 
-    MRRC(new RemoteRepository.Builder("mrrc", "default", "https://maven.repository.redhat.com/ga/")
+    MRRC(new RemoteRepository.Builder("mrrc", "default", "https://maven.repository.redhat.com/earlyaccess/all/")
```

Comment:
Now that the feature pack location are externalised, why do you still need to know these remote repositories?

*If* they stay, I think you should distinguish between `MRRC` (or `MRRC_GA`) and `MRRC_EA`

Reply:
Mostly they're useful in tests, I should just move them to test tree though